llmfoundry/data/finetuning/dataloader.py

```diff
@@ -275,12 +299,43 @@ def _validate_config(dataset_cfg: DictConfig) -> None:
             'dataset, but both were None.'
         )
 
+def _downloaded_datasets_dir() -> str:
```

Comment:
looks like this method get the directory path rather than downloading the dataset? Maybe a different method name?